tools/downloader/license.txt

```diff
@@ -2,1586 +2,15745 @@ Configuration file for the automation tools includes following models:
 
 ==================================================================================================
 
-* densenet-121, densenet-161, densenet-169, densenet-201 - Densely Connected Convolutional Networks  https://github.com/shicai/DenseNet-Caffe
```

Comment:
Could you group identical licenses, like they were grouped before? Otherwise, this file becomes quite unreadable, especially for Intel models.